### ISSUE TITLE
Swift Language Version Error

### ISSUE THREAD

**Issue by AlexHedley** [robbiehanson/KissXML#91]
I'm using [XMPPFramework](https://github.com/robbiehanson/XMPPFramework/) in an ObjC app and adding it to a new project I get the following error when compiling:

> “Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

Xcode 8.3.3

Is this a good workaround https://github.com/Alamofire/Alamofire/issues/1526

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_VERSION'] = '3.0'
    end
  end
end
```

**Comment by ric2z** [comment]
bump.

I'm having the same issue on Xcode 9.0. The suggested fix did **not** help (I'm trying to target `3.2`).

**Comment by l900416** [comment]
Similar issue.  when validating my cocoapods which has a dependency on  [XMPPFramework](https://github.com/robbiehanson/XMPPFramework) . 

```bash
=== BUILD TARGET KissXML OF PROJECT Pods WITH CONFIGURATION Release ===

Check dependencies
The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. This setting can be set in the build settings editor.
The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. This setting can be set in the build settings editor.

** BUILD FAILED **

The following build commands failed:
	Check dependencies
```

**Comment by ric2z** [comment]
@l900416 I was able to get this going by passing the Swift version on the terminal as well as specifying it on a `.swift-version` file.

Here's how you can specify it on the terminal:
`pod lib lint --swift-version=4.0`

I hope this helps.

**Comment by l900416** [comment]
@ric2z It works for me. thanks

**Comment by chrisballinger** [comment]
Swift is no longer required (resolved by #103)
